platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpoint.java

```diff
@@ -51,7 +52,7 @@
 
     private static final String DEFAULT_FILE_NAME = "file.jar";
 
-    private static String defaultFileLocation = "data/installer/uploads/";
+    private static String defaultFileLocation = System.getProperty("ddf.home") + "/data/installer/uploads/";
```

Comment:
While forward slashes will work cross-platform, it would be better to use `File.separator`. And it would be best to use [Paths.get()](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Paths.html#get-java.lang.String-java.lang.String...-).